.docs/Add-VSTeamTaskGroup.md

```diff
@@ -0,0 +1,83 @@
+<!-- #include "./common/header.md" -->
+
+# Add-VSTeamTaskGroup
+
+## SYNOPSIS
+
+<!-- #include "./synopsis/Add-VSTeamTaskGroup.md" -->
+
+## SYNTAX
+
+## DESCRIPTION
+
+<!-- #include "./synopsis/Add-VSTeamTaskGroup.md" -->
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
+
+```powershell
+
+$taskGroup = Get-VSTeamTaskGroup -Name "taskGroupName" -ProjectName "sourceProjectName"
+
+# Set the ID and revision to null, so AzD is happy.
+$taskGroup.id = $null
+$taskGroup.revision = $null
+
+$taskGroupJson = ConvertTo-Json -InputObject $taskGroup -Depth 10
+
+Add-VSTeamTaskGroup -Body $taskGroupJson -ProjectName "destinationProjectName"
+```
+
+## PARAMETERS
+
+### -ProjectName
+
+<!-- #include "./params/projectName.md" -->
+
+### -WhatIf
+
+Shows what would happen if the cmdlet runs.
+The cmdlet is not run.
+
+```yaml
+Type: SwitchParameter
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InFile
+
+The path to the json file that represents the task group
+
+### -Body
```

Comment:
Add the yaml section that shows parameter type and the parameter set name it is a part of and the fact it is required.